src/k8s-engine/lima.ts

```diff
@@ -811,15 +838,28 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
     });
   }
 
-  async start(config: { version: string; memoryInGB: number; numberCPUs: number; port: number; }): Promise<void> {
-    this.cfg = config;
+  // TODO: Put this in k3sHelper
+  createImageEventHandler(engineName: ContainerEngine) {
+    const imageProcessor = createImageProcessorFromEngineName(engineName, this);
+
+    if (!imageProcessor) {
+      throw new Error(`createImageEventHandler: No image processor for ${ engineName }`);
```

Comment:
Shouldn't `createImageProcessorFromEngineName` be the one to throw then?

Reply:
obsolete